istioctl/cmd/istioctl/main.go

```diff
@@ -298,8 +311,8 @@ istioctl get virtualservice bookinfo
 					strings.Join(supportedTypes(configClient), ", "))
 			}
 
-			typ, err := protoSchema(configClient, strings.ToLower(args[0]))
-			if err != nil {
+			typs, err := protoSchema(configClient, args[0], len(args) == 1)
```

Comment:
`protoSchema` shouldn't be overloaded to handle input/flag validation here. Lines 320-323 should catch the invalid allowAll/all combination.

Reply:
`kubectl get all --all-namespaces` works. Not sure why we're disallowing this combination for istioctl.